app/lib/preservation_catalog/aws/audit.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module PreservationCatalog
+  module AWS
+    # Methods for auditing the state of a ZippedMoabVersion on an AWS S3 endpoint.
+    class Audit < PreservationCatalog::S3Audit
+      delegate :bucket_name, to: ::PreservationCatalog::S3
+
+      private
+
+      def bucket
```

Comment:
Can this bucket method move to the superclass too?

Reply:
i like that idea, but the methods are slightly different at present, and while i'm sure they're consolidate-able, the exact way to do so doesn't jump out at me immediately.  since this PR as a whole seems like an improvement, could we do that move as follow-on work, and get this merged?

Reply:
alternatively, if the way to move this to the superclass is clear to you and i'm missing it, could you give just a bit more detail on how you'd abstract the differences?